docs/en/docs/tutorial/path-params-numeric-validations.md

```diff
@@ -106,9 +106,9 @@ And you can also declare numeric validations:
 * `le`: `l`ess than or `e`qual
 
 !!! info
-    `Query`, `Path`, and others you will see later are subclasses of a common `Param` class (that you don't need to use).
+    `Query`, `Path`, and others classes you will see later are subclasses of a common `Param` class.
 
-    And all of them share the same all these same parameters of additional validation and metadata you have seen.
+    All of them share the same parameters for additional validation and metadata.
```

Comment:
IIUC, everything under "Tutorial - User Guide" is intended to be read chronologically from top-to-bottom. In light of this intention, "you have seen" makes sense and I wouldn't remove it. I would add "so far in the tutorial" to make this intention explicit.

Reply:
Hey @paxcodes 👋 

I think these are great tweaks and I'll make sure to get them in. Thanks for the clarity.